### PR TITLE
add option to flag records that failed to Save in Data Refresh

### PR DESCRIPTION
**Pull request by brandon-klar**

### CODE REVIEW

**Comment by brandon-klar** [comment]
https://trello.com/c/Mdz7pZkI/15-add-an-option-to-data-refresh-to-flag-records-that-fail-on-save

File: skyve-ejb/src/main/java/modules/admin/DataMaintenance/RefreshDocumentTuplesJob.java [hunk 1]
```diff
@@ -89,6 +90,10 @@ public void execute() throws Exception {
 												sb.toString(),
 												dm.getRefreshOption().toLocalisedDescription(),
 												bean.getBizId()));
+						if (flagFailedData) {
```
Comment (benpetito) [inline skyve-ejb/src/main/java/modules/admin/DataMaintenance/RefreshDocumentTuplesJob.java]: Hi @brandon-klar, you should not perform a boolean operation on a Boolean object. This should be `if(Boolean.TRUE.equals(flagFailedData)) {`. Can you please update it?

**Review by benpetito — CHANGES_REQUESTED** [review]
Hi @brandon-klar , can you please see my comment on RefreshDocumentTuplesJob and address that?

**Comment by brandon-klar** [comment]
@benpetito - RefreshDocumentTuplesJob has been updated with if (Boolean.TRUE.equals(flagFailedData))

**Review by benpetito — APPROVED** [review]

**Pull request merged**